Server/Utils/vdb.py

```diff
@@ -57,9 +57,15 @@ def search_VDB_papers(self, query_embedding: List[float], top_k: int):
             data = [query_embedding],
             anns_field = "embedding",
             param = search_params,
+            output_fields=['title', 'abstract'],
             limit = top_k
         )
-        return results
+        # return is 
+        # ( int : dot product value , Entity : object)
+        # To get values from an Entity object
+        # EntityObject.get()
+        # ex: EntityObject.get("title"), EntityObject.get("abstract"), 
+        return results[0][0].distance, results[0][0].entity
 
 
     def Search_VDB_videos(self, query_embedding: List[float], top_k: int):
```

Comment:
in the definition of the method, if you specify the types of the input, you should specify the type of the output:
EX: def search(q : str) -> tuple[float, Dict]: